### DISCUSSION TITLE
You don't have the permission to access the requested resource. It is either read-protected or not readable by the server.

### DISCUSSION THREAD

**Discussion by boboboss** [langgenius/dify#7566] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

After deployment, those who do not have administrator privileges cannot view the logs and annotations they created.

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by sukai-cheng** [comment]
+1

**Comment by sukai-cheng** [comment]
添加这一步支持外部访问
<img width="1053" alt="image" src="https://github.com/user-attachments/assets/6433c5b8-0da6-4d38-8706-e78670784f49" />

**Comment by chk4991** [comment]
请问解决了吗？
